tests/sentry/notifications/utils/test_should_be_notified.py

```diff
@@ -0,0 +1,37 @@
+from unittest import TestCase
+
+from sentry.models import User
+from sentry.notifications.helpers import where_should_user_be_notified
+from sentry.notifications.types import NotificationScopeType, NotificationSettingOptionValues
+from sentry.types.integrations import ExternalProviders
+
+
+class WhereShouldBeNotifiedTest(TestCase):
+    def setUp(self) -> None:
+        self.user = User(id=1)
```

Comment:
I think I might have asked this elsewhere before, but why use this instead of the factory method to create a user?

Reply:
For faster tests: https://github.com/getsentry/sentry/pull/26756#discussion_r655793228